app/celery/process_ga4_measurement_tasks.py

```diff
@@ -72,6 +55,18 @@ def post_to_ga4(
         current_app.logger.error('GA4_MEASUREMENT_ID is not set')
         return False
 
+    # Retrieve the notification from the database
+    stmt = select(Notification).where(Notification.id == notification_id)
+    notification = db.session.scalars(stmt).first()
+    if not notification:
+        current_app.logger.error('Notification %s not found', notification_id)
```

Comment:
Could we update this statement to say something about ga4 so we know where it's coming from in the logs?

Reply:
Yes, for sure, all the others have GA4 which is nice